frontend/src/scenes/debug/Modifiers.tsx

```diff
@@ -99,6 +99,22 @@ export function Modifiers<Q extends { response?: Record<string, any>; modifiers?
                     value={query.modifiers?.materializationMode ?? response?.modifiers?.materializationMode}
                 />
             </LemonLabel>
+            <LemonLabel className={labelClassName}>
+                <div>Optimize joined filters:</div>
+                <LemonSelect
```

Comment:
This should probably have a default, because currently only "Select a value" is shown when unset

Reply:
The modifiers on the debug page are supposed to be empty if not explicitly set, or received from the API. The defaults here are set on the API side, so the idea is that you'll see them when the query returns.